usecases/scheduler/src/lambda/ssv2_resultProcesser/resultProcesser.py

```diff
@@ -0,0 +1,263 @@
+import boto3 
+import json
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import os
+import openpyxl
+import logging
+
+## Initialization
+SnsEmailSubject = "[AWS] Service Screener Scheduler Report"
+SnsEmailBodyPrefix = "Here is your Service Screener report.\n\n"
+snsPrefix = os.environ['SSV2_SNSARN_PREFIX']
+
+
+successResp = {
+    'statusCode': 200,
+    'body': 'success'
+}
+
+currentResults = {}
+previousResults = {}
+accounts = {}
+
+SHEETS_TO_SKIP = ['Info', 'Appendix']
+
+def lambda_handler(event, context):
+    record = event['detail']
+    region = event['region']
+    targetBucket = record['bucket']['name']
+    targetObject = record['object']['key']
+    
+    s3 = boto3.client('s3', region_name=region)
+    sns = boto3.client('sns', region_name=region)
+    
+    filepath = targetObject.split('/')
+    configId = filepath[0]
+    currentDate = filepath[1]
+
+    objs = s3.list_objects_v2(Bucket=targetBucket, Prefix=configId + '/' + currentDate + '/')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `list_objects_v2` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `ContinuationToken`, `IsTruncated`, `NextContinuationToken`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)